glean-core/ios/Glean.xcodeproj/project.pbxproj

```diff
@@ -9,9 +9,6 @@
 /* Begin PBXBuildFile section */
 		BF3DE39B2243A2F20018E23F /* Glean.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = BF3DE3912243A2F20018E23F /* Glean.framework */; };
 		BF3DE3A02243A2F20018E23F /* GleanTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = BF3DE39F2243A2F20018E23F /* GleanTests.swift */; };
-		BF51C3BD224BF19F0014CBAF /* debug.xcconfig in Resources */ = {isa = PBXBuildFile; fileRef = BF51C3B9224BF19F0014CBAF /* debug.xcconfig */; };
```

Comment:
Oh wait, does this mean these are not used anymore?
Hm, the project still builds in its default configuration now, but maybe we need to bring them bake later (I would check against the a-s codebase and see what they do these days)